e2e/testdata/fn-eval/fn-source/exclude-meta-resources-by-default/.expected/exec.sh

```diff
@@ -0,0 +1,20 @@
+#! /bin/bash
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eo pipefail
+
+kpt fn source \
+| kpt fn eval - --image gcr.io/kpt-fn/set-namespace:v0.1.3 -- namespace=staging \
+| kpt fn sink .
```

Comment:
you don't need the '.' , it's the default

Reply:
`DIR` is required now as mentioned in https://github.com/GoogleContainerTools/kpt/pull/2032#discussion_r635658705.

Create https://github.com/GoogleContainerTools/kpt/issues/2159 to track it.